### PR TITLE
Enhancement: run humbug in the test suite

### PR DESCRIPTION
**Pull request by Ocramius**
This PR adds humbug to the test suite - no more wobbly tests allowed

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
     },
     "require-dev": {
         "phpunit/phpunit":              "^5.0",
-        "zendframework/zend-diactoros": "^1.1"
+        "zendframework/zend-diactoros": "^1.1",
+        "humbug/humbug":                "dev-master"
```
Comment (Ocramius) [inline composer.json]: There's no stable version, sorry :-\

**Comment by Ocramius** [comment]
Ok, fails because of no XDebug on 7.1... not sure how to fix that :X

**Comment by necromant2005** [comment]
let's add hack for php 7.1
if php -i |grep -qE xdebug; then ./vendor/bin/humbug; fi

I've checked it works well on php 7.1 https://travis-ci.org/necromant2005/storageless/builds/159680596

**Comment by Ocramius** [comment]
Good hint!

**Comment by lcobucci** [comment]
@Ocramius I'll reorganize the commits a bit

**Comment by Ocramius** [comment]
@lcobucci I'm kinda against squashing <_<

**Comment by lcobucci** [comment]
@Ocramius alright I'll keep my OCD out of this repo for now 😛

**Pull request merged**